server/service/statestore.go

```diff
@@ -263,6 +263,10 @@ func (store *stateStore) Save() error {
 	return nil
 }
 
+// Start the state store. If the state store is configured to stop after initializing, it
+// will exit when it is initialized, or if it hits a timeout. Otherwise it will run as a
+// daemonized process and send pings to keep the raft alive. Errors encountered while running
+// will be sent into errchan. startedC is used to start the raft.
```

Comment:
I think I could use a better description here, especially as to the purpose of ```startedC```. Suggestions?

Reply:
`startedC will be closed when the Raft node has been initialized and will start to perform serving. This can be used to wait for serving publicly until the internals are available.`

Reply:
Excellent, I'm applying this patch:

```
diff --git a/server/service/statestore.go b/server/service/statestore.go
index 896bf40..8a42f86 100644
--- a/server/service/statestore.go
+++ b/server/service/statestore.go
@@ -266,7 +266,9 @@ func (store *stateStore) Save() error {
 // Start the state store. If the state store is configured to stop after initializing, it
 // will exit when it is initialized, or if it hits a timeout. Otherwise it will run as a
 // daemonized process and send pings to keep the raft alive. Errors encountered while running
-// will be sent into errchan. startedC is used to start the raft.
+// will be sent into errchan. startedC will be closed when the Raft node has been initialized and
+// will start serving. This can be used to wait for serving publicly until the internals are
+// available.
 func (store *stateStore) RunStateStore(errchan chan<- error, startedC chan<- struct{}) {
        // First start the raft node
        go store.raftnode.startRaft(startedC)
```